### ISSUE TITLE
Is there any sticky header feature like sectionHeadersPinToVisibleBounds in UICollectionViewFlowLayout?

### ISSUE THREAD

**Issue by gardenrabbit** [patchthecode/JTAppleCalendar#467]
I am customizing my calendar view with version 7.0.4.
I want to make month headers to stick to the top of the view bounds.
I am considering getting calendar layout to subclass UICollectionViewFlowLayout.
Is there any other way?

**Comment by patchthecode** [comment]
hmm.. i cant visualize what you are asking. Got an image?

> I want to make month headers to stick to the top of the view bounds.

**Comment by patchthecode** [comment]
Hmm.. you know.. i never even know this about flowLayout haha. No, i havent coded this into this library. But I can think about it for an enhancement some time in future. Or someone can do a pull request.

**Comment by Ayal-Spitz-Intrepid** [comment]
@patchthecode did you ever implement this feature?

**Comment by hoollyby** [comment]
+1

**Comment by patchthecode** [comment]
no. i didnt. i'll have to let someone else implement that unfortunately.
ive work on this lib for quite a long time, and i consider it complete for my purposes.
i've packed it with every feature imaginable for its time. If someone wants to do this and implement it i'll take a look
